src/model/Card.js

```diff
@@ -1,10 +1,8 @@
-'use strict';
-
 /** Represents bank card
  *
  * @property {number} cardId - Card's id in Lemon DB.
  * @property {string} bank - Name of the bank card belongs to
- * @property {number} cardNum - Card number (can be either all number or last 4 digits only)
+ * @property {number} cardNum - Card number (must be full number)
```

Comment:
Why so?

Reply:
Because Privat API needs full number of card (check data sample in privat24.js)

Reply:
But other API's don't. And don't provide it.